package.json

```diff
@@ -19,15 +19,17 @@
     "build:types": "tsc --declaration --emitDeclarationOnly --declarationDir ./dist",
     "build:api_docs": "jsdoc2md > API.md",
     "pretest": "npm run prettier && npm run typecheck",
-    "test": "vitest run",
+    "test": "vitest run signer",
+    "test:uploads": "vitest run actual-uploads",
```

Comment:
Ideally have `test:uploads` run in CI so it actually gets used, otherwise folks will forget to run it.

Reply:
put in #10